var/spack/repos/builtin/packages/ghostscript/package.py

```diff
@@ -29,14 +29,23 @@ class Ghostscript(Package):
     """An interpreter for the PostScript language and for PDF."""
 
     homepage = "http://ghostscript.com/"
-    url = "http://downloads.ghostscript.com/public/old-gs-releases/ghostscript-9.18.tar.gz"
 
+    version('9.21', '6f60d7fcb5eef6a8bec5abedf21c6a7008a8c0c7')
     version('9.18', '33a47567d7a591c00a253caddd12a88a')
 
     parallel = False
 
     depends_on('libtiff')
 
+    def url_for_version(self, version):
```

Comment:
Do you know for sure that `version > 9.18` is the correct criterion here?  What about versions `9.19` and `9.20`?

Can `url_for_version()` only implement the old URL?  I'm thinking something like this:
```
   def url_for_version(self, version):
        if version <= Version('9.18'):
            return "http://downloads.ghostscript.com/public/old-gs-releases/ghostscript-{0}.tar.gz" \
                .format(version.dotted)
    return super().url_for_version(version) 
```
[I used Python3 syntax here for super(); this will need Python2 syntax].